core/src/test/java/com/linecorp/armeria/common/util/CompositeExceptionTest.java

```diff
@@ -0,0 +1,235 @@
+    }
+
+    @Test
+    public void paddingTest() {
+        final IllegalStateException ex1 = new IllegalStateException();
+        final IllegalArgumentException ex2 = new IllegalArgumentException();
+        final String className = getClass().getSimpleName();
+        final String methodName = "verboseExceptionDisabledTestIfStackTraceLengthLessThan20";
+        final String fileName = "CompositeExceptionTest.java";
+        final StackTraceElement[] customStackTrace = {
+                new StackTraceElement(className, methodName, fileName, 1),
+                new StackTraceElement(className, methodName, fileName, 2),
+                new StackTraceElement(className, methodName, fileName, 3),
+        };
+        ex1.setStackTrace(customStackTrace);
+        ex2.setStackTrace(customStackTrace);
+        final CompositeException compositeException =
+                new CompositeException(ImmutableList.of(ex1, ex2), Sampler.never());
+        final int paddingCount = paddingCount(compositeException);
+
+        // padding * stack trace length * exception length
+        final int expectPaddingCount = 4 * 3 * 2;
+        assertThat(paddingCount).isEqualTo(expectPaddingCount);
+    }
+
+    @Test
+    public void compositeExceptionPaddingTest() {
+        final IllegalStateException ex1 = new IllegalStateException();
+        final IllegalArgumentException ex2 = new IllegalArgumentException();
+        final AnticipatedException ex3 = new AnticipatedException(ex2);
+        final String className = getClass().getSimpleName();
+        final String methodName = "verboseExceptionDisabledTestIfStackTraceLengthLessThan20";
+        final String fileName = "CompositeExceptionTest.java";
+        final StackTraceElement[] customStackTrace = {
+                new StackTraceElement(className, methodName, fileName, 1),
+                new StackTraceElement(className, methodName, fileName, 2),
+                new StackTraceElement(className, methodName, fileName, 3),
+        };
+        ex1.setStackTrace(customStackTrace);
+        ex2.setStackTrace(customStackTrace);
+        ex3.setStackTrace(customStackTrace);
+        final CompositeException compositeException =
+                new CompositeException(ImmutableList.of(ex1, ex3), Sampler.never());
+        final int paddingCount = paddingCount(compositeException);
+        // padding * custom stack trace length * exception length + ( composite exception padding *
+        // custom stack trace length )
+        final int expectPaddingCount = 4 * 3 * 2 + (6 * 3);
+        assertThat(paddingCount).isEqualTo(expectPaddingCount);
+    }
+
+    private static long stackTraceLength(final CompositeException exception) {
+        return Arrays.stream(exception.getCause().getMessage().split(separator))
+                        .filter(line -> !line.contains("Multiple exceptions") && !line.contains("|-- "))
+                        .count();
+    }
```

Comment:
Please check the indentation. 😉 
https://armeria.dev/community/developer-guide#setting-up-your-ide
```suggestion
        return Arrays.stream(exception.getCause().getMessage().split(separator))
                     .filter(line -> !line.contains("Multiple exceptions") && !line.contains("|-- "))
                     .count();
    }
```